### DISCUSSION TITLE
Bootstrap 5

### DISCUSSION THREAD

**Discussion by scottkuhl** [PeterHimschoot/microsoft-blazor-book-3#4] [General]
From Chapter 2:

_Out of the box, Blazor uses the popular Bootstrap 4 layout framework (https://getbootstrap.com/), including open-iconic fonts._

The .NET 6 templates use Boostrap 5.  This looks like a minor typo leftover from the previous edition.

**Comment by scottkuhl** [comment]
This appears in Chapter 3 as well.

_Note: The default Blazor templates use Bootstrap 4 for styling. Bootstrap (http://getbootstrap.com) is a very popular CSS framework, originally built for Twitter, giving an easy layout for web pages._

**Reply by PeterHimschoot** [reply]
Actually, at time of writing it was still bootstrap 4, but when they released .NET 6 RTM Microsoft switched to bootstrap 5 and the manuscipt was ready by then :)

Blazor does not care which CSS framework you use, you could for example use TailwindCSS (https://tailwindcss.com/).

**Reply by PeterHimschoot** [reply]
I even had to update PizzaPlace to use different css classes. Pull this repository from time to time, I am still working on updating some of the sample code.

**Comment by Ryhertzy** [comment]
Along these same lines, I think the pricing columns in chapter 2 should be "text-end" instead of "text-right".
